astropy/coordinates/tests/test_frames.py

```diff
@@ -753,12 +753,12 @@ def test_time_inputs():
         c = FK4(1 * u.deg, 2 * u.deg, obstime="hello")
     assert "Invalid time input" in str(err.value)
 
-    # A vector time should work if the shapes match, but we don't automatically
-    # broadcast the basic data (just like time).
-    FK4([1, 2] * u.deg, [2, 3] * u.deg, obstime=["J2000", "J2001"])
-    with pytest.raises(ValueError) as err:
-        FK4(1 * u.deg, 2 * u.deg, obstime=["J2000", "J2001"])
-    assert "shape" in str(err.value)
+    # A vector time should work if the shapes match, and we automatically
+    # broadcast the basic data.
+    c = FK4([1, 2] * u.deg, [2, 3] * u.deg, obstime=["J2000", "J2001"])
+    assert c.shape == (2,)
+    c = FK4(1 * u.deg, 2 * u.deg, obstime=["J2000", "J2001"])
```

Comment:
This used to throw error but not anymore. Will backporting this be too surprising to users?

Reply:
OK, I think I'm getting the message "feature, not bug" from both of you, @pllim and @mhvk. Would you like me to switch the change log entry back to "feature"?

Side question, do you have a sense of when astropy 6 will be released?